iceoryx_hoofs/BUILD.bazel

```diff
@@ -59,13 +60,14 @@ cc_library(
         "utility/source/*.cpp",
         "vocabulary/source/**/*.cpp",
     ]),
-    hdrs = glob(["buffer/**"]) + glob(["cli/**"]) + glob(["container/**"]) + glob(["design/**"]) + glob(["filesystem/**"]) + glob(["functional/**"]) + glob(["include/**"]) + glob(["legacy/**"]) + glob(["memory/**"]) + glob(["posix/**"]) + glob(["primitives/**"]) + glob(["reporting/**"]) + glob(["time/**"]) + glob(["utility/**"]) + glob(["vocabulary/**"]) + [
+    hdrs = glob(["buffer/**"]) + glob(["cli/**"]) + glob(["concurrent/**"]) + glob(["container/**"]) + glob(["design/**"]) + glob(["filesystem/**"]) + glob(["functional/**"]) + glob(["include/**"]) + glob(["legacy/**"]) + glob(["memory/**"]) + glob(["posix/**"]) + glob(["primitives/**"]) + glob(["reporting/**"]) + glob(["time/**"]) + glob(["utility/**"]) + glob(["vocabulary/**"]) + [
```

Comment:
Could we reorder this so that every `glob([...]) +` entry has its own line?

Reply:
AFAIK the Bazel linter does not allow that or we would need to overwrite the default somehow.

Reply:
It seems bazel does not like to have that many subfolders. Unfortunately I also do not know how to change the defaults. There is also the issue that the Bazel 7 linter breaks current master